### PR TITLE
[FIX] Spelling #1

### PR DESCRIPTION
**Pull request by oliveralka**
# Description

One part of the fixes for spelling

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Review by jpfeuffer — APPROVED** [review]

**Comment by oliveralka** [comment]
Failing win test is unrelated:

```
checking EmpiricalFormula& operator<(const EmpiricalFormula& rhs) ... 
 +  line 138:  TEST_EQUAL(EmpiricalFormula("C5H2") < EmpiricalFormula("C6H2"),true): got '1', expected '1'
 +  line 139:  TEST_EQUAL(EmpiricalFormula("C5H2") < EmpiricalFormula("C5H3"),true): got '1', expected '1'
 +  line 140:  TEST_EQUAL(EmpiricalFormula("C5") < EmpiricalFormula("C5H2"),true): got '1', expected '1'
 +  line 142:  TEST_EQUAL(EmpiricalFormula("C5H2") < EmpiricalFormula("C4H2"),false): got '0', expected '0'
 +  line 143:  TEST_EQUAL(EmpiricalFormula("C5") < EmpiricalFormula("C5"),false): got '0', expected '0'
 -  line 145:  TEST_EQUAL(EmpiricalFormula("C5H2") < EmpiricalFormula("C5D2"),true): got '0', expected '1'
 +  line 146:  TEST_EQUAL(EmpiricalFormula("C5D2") < EmpiricalFormula("C5T2"),true): got '1', expected '1'
: failed
```

File: src/openms/source/ANALYSIS/ID/PeptideProteinResolution.cpp [hunk 1]
```diff
@@ -713,7 +713,7 @@ namespace OpenMS
     size_t best_grp_index = *conn_comp.prot_grp_indices.begin();
     ambiguity_grp.probability = origin_groups[best_grp_index].probability;
     
-    // copy group indices so we can reasily reorder them for tie resolution
+    // copy group indices so we can  reorder them for tie resolution
```
Comment (timosachsenberg) [inline src/openms/source/ANALYSIS/ID/PeptideProteinResolution.cpp:716]:
```suggestion
    // copy group indices so we can reorder them for tie resolution
```

**Review by timosachsenberg — APPROVED** [review]

**Comment by timosachsenberg** [comment]
nice! thanks

**Comment by oliveralka** [comment]
Windows error was fixed in https://github.com/OpenMS/OpenMS/pull/5462

**Pull request merged**